coalib/processes/Processing.py

```diff
@@ -430,6 +431,14 @@ def get_ignore_scope(line, keyword):
         return list(StringConverter(toignore, list_delimiters=', '))
 
 
+def get_noqa_scope(line, keyword):
+    toignore = line[line.rfind(keyword) + len(keyword):]
+    if toignore.startswith('all'):
```

Comment:
I need some help here. `toignore.startswith(''): ` doesn't do the job; but it needs to check if there is nothing after the word 'noqa'.